sys/net/netif/netif.c

```diff
@@ -74,6 +75,11 @@ netif_t *netif_get_by_name(const char *name)
     return NULL;
 }
 
+netif_t *netif_get_by_name(const char *name)
+{
+    return netif_get_by_name_buffer(name, strlen(name));
+}
```

Comment:
This can be `static inline` in the header now.

Reply:
Moved to the header